app/models/manual_record.rb

```diff
@@ -12,7 +12,7 @@ class ManualRecord
     autosave: true
 
   def self.find_by(attributes)
-    first(conditions: attributes)
+    where(attributes).first
```

Comment:
Great! I'm going to assume that we'll notice this again in future so am not going to do anything about it at the moment.